11-checkout.md

```diff
@@ -1,119 +0,0 @@
----
-layout: lesson
-subtitle: Instructor Checkout Procedure
----
-> ## Motivating Question
-> 
-> What are the final steps in becoming an instructor trainer?
-{: .question}
-
-The final steps in getting certified are as follows:
-
-1. Pick one of the lessons we are currently accepting trainee requests on
-   and submit a *small* change.
-   (Your trainer will provide you with a list of lessons.)
-   If you want to teach Software Carpentry,
-   you must submit the change as a pull request on GitHub
-   and send the link to your trainer.
-   If you want to teach Data Carpentry,
-   go to [its submission page][dc-submission-page]
-   (which will redirect you to the appropriate place).
-
-   Note:
```

Comment:
Following the comment from @ChristinaLK on cutting down on text, I'd say that this note could possibly be removed or rather moved to the materials for the trainers. They need to know this and explain, if needs be.